lib/src/main/java/com/team2813/lib2813/util/ControlUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.team2813.lib2813.util;
+
+/**
+ * Utility class for comon control-related utility functions.
+ */
+public class ControlUtils {
+    private ControlUtils() {
+        throw new AssertionError("Cannot create ControlUtils instance. Use its static methods directly.");
+    }
+
+    /**
+     * Deadbands a value.
+     * 
+     * "A deadband or dead-band (also known as a dead zone or a neutral zone) is a
+     * band of input values in the domain of a transfer function in a control system
+     * or signal processing system where the output is zero (the output is 'dead' -
+     * no action occurs)." https://en.wikipedia.org/wiki/Deadband
```

Comment:
This link will not be clickable when the javadoc html is generated, or with IntelliJ/VS Code hover text.
```suggestion
     * no action occurs)." <a href="https://en.wikipedia.org/wiki/Deadband">https://en.wikipedia.org/wiki/Deadband</a>
```